spec/models/partnership_spec.rb

```diff
@@ -3,24 +3,4 @@
 require "rails_helper"
 
 RSpec.describe Partnership, type: :model do
-  it "can be unconfirmed" do
-    partnership = Partnership.create!(
-      school: FactoryBot.create(:school),
-      lead_provider: FactoryBot.create(:lead_provider),
-    )
-    expect(partnership.confirmed?).to be false
-  end
-
-  it "can be confirmed" do
-    partnership = Partnership.create!(
-      school: FactoryBot.create(:school),
-      lead_provider: FactoryBot.create(:lead_provider),
-    )
-
-    partnership.confirm
-    expect(partnership.confirmed?).to be true
-
-    database_partnership = Partnership.first
-    expect(database_partnership.confirmed?).to be true
-  end
 end
```

Comment:
This is a sad, sad file 😂 

Is there anything we want to test here?

Reply:
I was adding
```
describe "associations" do
    it { is_expected.to belong_to(:school) }
    it { is_expected.to belong_to(:lead_provider) }
    it { is_expected.to belong_to(:cohort) }
end
```
on my branch

Reply:
Added and stolen from Ethan

Reply:
Done